### PR TITLE
Enable translate in RELEASE 25%

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-variations/issues/335
PR to main: https://github.com/brave/brave-variations/pull/343

### CODE REVIEW

**Review by iefremov — APPROVED** [review]

**Review by linhkikuchi — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `main` via https://github.com/brave/brave-variations/pull/343#issuecomment-1234350258.

**Pull request merged**
